src/executeCommandLine/executeCommandLine.ts

```diff
@@ -616,9 +618,21 @@ namespace ts {
         }
     }
 
-    function reportStatistics(sys: System, program: Program) {
+    function reportStatistics(sys: System, program: Program, reportDiagnostic: DiagnosticReporter) {
         let statistics: Statistic[];
         const compilerOptions = program.getCompilerOptions();
+
+        if (compilerOptions.expensiveStatements) {

```

Comment:
They're really only diagnostics for the pretty-printing.  Otherwise, they seem like they belong with the statistics.